app/views/users/show.html.erb

```diff
@@ -1,2 +1,80 @@
-<h1><%= @user.name %></h1>
-<!-- Other user details -->
+<!DOCTYPE html>
+<html>
+<head>
+  <title><%= @user.name %></title>
+  <style>
+    h1 {
+      color: #333;
+      font-size: 32px;
+      margin-bottom: 20px;
+    }
+
+    p {
+      color: #666;
+      font-size: 18px;
+      margin-bottom: 10px;
+    }
+
+    img {
+      max-width: 300px;
+      max-height: 300px;
+      border-radius: 50%;
+      margin-bottom: 20px;
+    }
+
+    .post-container {
+      border: 1px solid #ccc;
+      padding: 10px;
+      margin-bottom: 20px;
+    }
+
+    .post-container h2 {
+      font-size: 24px;
+      margin-bottom: 10px;
+    }
+
+    .post-container p {
+      font-size: 16px;
+      margin-bottom: 5px;
+    }
+
+    .comments-list {
+      list-style: none;
+      margin-left: 20px;
+    }
+
+    .comments-list li {
+      margin-bottom: 5px;
+    }
+
+    .comments-list li span {
+      font-weight: bold;
+    }
+  </style>
+</head>
+<body>
+  <h1><%= @user.name %></h1>
+  <% if @user.photo.present? %>
+    <%= image_tag @user.photo %>
+  <% else %>
+    <p>No photo available</p>
+  <% end %>
+  <p><%= @user.bio %></p>
+  <p>Number of posts: <%= @user.posts.count %></p>
+
+  <% @user.posts.limit(3).each_with_index do |post, index| %>
+    <div class="post-container">
+      <h2>Post #<%= index + 1 %> by <%= post.author.name %></h2>
+      <p><%= post.text %></p>
+      <p>Comments: <%= post.comments.count %>, Likes: <%= post.likes.count %></p>

```

Comment:
- [ ] As raised earlier, the `see all post button` is missing from the UI, kindly implement this for you to move ahead. 🙏